src/AppMenuButtonGroup.cc

```
@@ -363,9 +367,10 @@ void AppMenuButtonGroup::resetButtons()
     }
     setCurrentIndex(-1);
     m_currentMenu = nullptr;
-    m_lastResults.clear();
-    m_lastSearchQuery.clear();
-    m_actionTextCache.clear();
+    if (m_search) {
```

Clang (14.0.6)

[warning] implicit conversion 'Material::AppMenuSearch *' -> bool

(readability-implicit-bool-conversion)